src/Dispatcher/DecoratedDispatcher.php

```diff
@@ -52,30 +46,31 @@ public function dispatch(ServerRequestInterface $request, ResponseInterface $res
 			$request = $e->getRequest();
 			$response = $e->getResponse();
 			$e = $e->getPrevious();
-		} catch (Exception $e) {
+		} catch (Throwable $e) {
 			// Process exception in the next lines
 		}
 
 		if (isset($e)) {
-			// Trigger exception decorator
-			$response = $this->decoratorManager->decorateResponse(IDecorator::ON_DISPATCHER_EXCEPTION, $request, $response, ['exception' => $e]);
-
-			// If there's no decorator to handle this exception, throw again
-			if ($response === NULL) throw $e;
+			if ($this->decoratorManager->hasDecorators(IDecorator::ON_DISPATCHER_EXCEPTION)) {
+				// Trigger exception decorator
```

Comment:
Takhle to nepujde, ono se muze stat, ze tam decorator je, ale ze vrati null, protoze s danou exception nic nedela. Ten puvodni kod byl dobre.